src/main/resources/application-local.yaml

```diff
@@ -0,0 +1,22 @@
+spring:
+  application:
+    name: stockanalyzer
+  data:
+    mongodb:
+      uri: mongodb://root:secret@localhost:27017/stocks?authSource=admin
+api:
+  finnhub:
+    token: ${FINNHUB_API_TOKEN}
+    symbols: SPUP,BNRG,HIG.PRG,RILYT,GBARF,WINN,PWWBF,RHBAF,CLVR,UPLD,AJGFF,HLLY,MSIU,ALNKF,UCRD,MGNT,IMAQ,SWRBF,BAYRY,RVHCS,APTPF,LALFS,HIVE,FNLIF,TLIS,CRTO,ORGO,OZBKF,UNIKF,DSPHF,FRCLL,CLMB,NCXS,GXRFF,JL,AMSRF,BITC,TLSNY,TGMR,TZPC,ADDHY,CBVTF,GIGM,CIBR,AFSIM,GULDF,CSBF,LFMDP,ABMC,LVWR.WS,PDEX,BFS.PRE
```

Comment:
it duplicates property from `application.yaml`. 
Maybe, we should delete `symbols` here?